### ISSUE TITLE
Nightmare Creatures 2

### ISSUE THREAD

**Issue by wicksand420** [Rinnegatamante/flycast-vita-Compatibility#167]
# Game summary
- Game name: Nightmare Creatures 2
- Game version: cdi usa

# flycast summary
- Version: v.0.1

# Issues

Crashes directly on startup Pre-bios
# Screenshots
![](https://?)

# Recommended labels

- Crash

**Comment by Rinnegatamante** [comment]
SMC Accuracy and SMC Performance?

**Comment by wicksand420** [comment]
Yes, tested all 3, same, crash before bios, this may be a issue with the cdi I have, later I will be testing the gdi versions of all the games I did labeled "crash" to see if they work, because i feel most of these may be a problem with the way someone dumped the game, especially the ones that crash before the bios screen.

**Comment by wicksand420** [comment]
Gdi does the same with all 3 cores
